server/internal/usecase/pagination.go

```diff
@@ -0,0 +1,76 @@
+package usecase
+
+type Cursor string
+
+type Pagination struct {
+	Before *Cursor
+	After  *Cursor
+	First  *int
+	Last   *int
+}
+
+//func NewPagination(first *int, last *int, before *Cursor, after *Cursor) *Pagination {
+//	// Relay-Style Cursor Pagination
+//	// ref: https://www.apollographql.com/docs/react/features/pagination/#relay-style-cursor-pagination
+//	return &Pagination{
+//		Before: before,
+//		After:  after,
+//		First:  first,
+//		Last:   last,
+//	}
+//}
```

Comment:
```suggestion
// Pagination is a struct for Relay-Style Cursor Pagination
// ref: https://www.apollographql.com/docs/react/features/pagination/#relay-style-cursor-pagination
type Pagination struct {
	Before *Cursor
	After  *Cursor
	First  *int
	Last   *int
}
```

Remember that public functions in Go do not need to be commented out because it is not an error if they are not used. Nevertheless, this function does not do anything meaningful, so you can delete it.